packages/custom-elements/tsconfig.json

```diff
@@ -1,15 +1,14 @@
 {
   "compilerOptions": {
```

Comment:
The last property of `compilerOptions` has a trailing comma.

Reply:
Fixed